Mage/src/main/java/mage/game/permanent/token/OminousRoostToken.java

```diff
@@ -0,0 +1,32 @@
+package mage.game.permanent.token;
+
+import mage.MageInt;
+import mage.abilities.common.CanBlockOnlyFlyingAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.keyword.FlyingAbility;
+import mage.constants.CardType;
+import mage.constants.SubType;
+
+public class OminousRoostToken extends TokenImpl {
+    public OminousRoostToken() {
+        super("Bird", "blue Bird creature token with flying and " +
+                "\"This creature can block only creatures with flying.\"");
+        cardType.add(CardType.CREATURE);
+        color.setBlue(true);
+        subtype.add(SubType.BIRD);
+        power = new MageInt(1);
+        toughness = new MageInt(1);
+        
+        this.addAbility(FlyingAbility.getInstance());
+        this.addAbility(new SimpleStaticAbility(new CanBlockOnlyFlyingAbility()));
```

Comment:
You can just put `this.addAbility(new CanBlockOnlyFlyingAbility());`, you don't need the `SimpleStaticAbility`

Reply:
This is my first time working with java so i hope i don't make too many mistakes and that all new implementations are ok, still have to get used to the syntax, the repo and where / how things are implemented. ^^